src/fedora/27/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM fedora:27
+
+# Workaround to avoid NuGet restore timeout
+RUN dnf upgrade -y nss \
+    && dnf clean all
+
+# Install the base toolchain we need to build anything (clang, cmake, make and the like)
+# this does not include libraries that we need to compile different projects, we'd like
+# them in a different layer.
+RUN dnf install -y \
+        clang \
+        cmake \
+        lldb-devel \
+        llvm-devel \
+        make \
+        python \
+        which \
+    && dnf clean all
+
+# Install tools used by the VSO build automation.
+RUN dnf install -y \
+        git \
+        npm \
+        nodejs \
+        tar \
+        zip \
+    && dnf clean all
+
+RUN npm install -g azure-cli --unsafe-perm
+
+# Dependencies of CoreCLR and CoreFX.
+RUN dnf install -y \
+        glibc-locale-source \
+        libcurl-devel \
+        libgdiplus \
+        libicu-devel \
+        libunwind-devel \
+        libuuid-devel \
+        lttng-ust-devel \
+        openssl-devel \
+        uuid-devel \
+    && dnf clean all
+
+RUN dnf install -y compat-openssl10-devel --allowerasing
+
+# Until Fedora.27-x64 official packages are available, we have to restore the ubuntu ones instead.
+ENV __PUBLISH_RID=ubuntu.14.04-x64
```

Comment:
Can this set in the `docker run` cmd so that the Docker image does not have to be rebuilt and all of the build definitions updated once the packages are available?